### PR TITLE
Allow blob sas uri from file as input

### PR DESCRIPTION
**Pull request by anusapan**
---
This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

Thank you for contributing to the IoT extension!

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] If introducing new functionality or modified behavior, are they backed by unit and integration tests?
- [ ] In the same context as above are command names and their parameter definitions accurate? Do help docs have sufficient content?
- [ ] Have **all** unit **and** integration tests passed locally? i.e. `pytest <project root> -vv`
- [ ] Have static checks passed using the .pylintrc and .flake8 rules? Look at the CI scripts for example usage.

### CODE REVIEW

**Comment by anusapan** [comment]
Integration Test

image

**Comment by anusapan** [comment]
Unit test

image

File: azext_iot/_params.py [hunk 1]
```diff
@@ -382,7 +382,7 @@ def load_arguments(self, _):
             help="Blob Shared Access Signature URI with write, read, and delete access to "
             "a blob container. This is used to output the status of the "
             "job and the results. Note: when using Identity-based authentication, you must supply an "
```
Comment (digimaun) [inline azext_iot/_params.py:384]: I think this makes the last part of the text more clear (same application for all the descriptions) 

"Note: when using Identity-based authentication an https:// URI is still required. Input for this argument can be inline or from a file path."

File: azext_iot/_help.py [hunk 2]
```diff
@@ -197,6 +197,13 @@
     short-summary: Export all device identities from an IoT Hub to an Azure Storage blob container.
     long-summary: For more information, see
                   https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities
+    examples:
+    - name: Export all device identities to blob container using SAS uri.
+      text: >
+        az iot hub device-identity export -n {iothub_name} --bcu {sas_uri}
+    - name: Export all device identities to blob container using file which contains Blob SAS uri.
```
Comment (digimaun) [inline azext_iot/_help.py:204]: super nit: can we lowercase "Blob" near the end

File: azext_iot/_help.py [hunk 3]
```diff
@@ -197,6 +197,13 @@
     short-summary: Export all device identities from an IoT Hub to an Azure Storage blob container.
     long-summary: For more information, see
                   https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities
+    examples:
+    - name: Export all device identities to blob container using SAS uri.
```
Comment (digimaun) [inline azext_iot/_help.py:201]: "Export all device identities to a configured blob container using an inline SAS uri."

**Pull request merged**